packages-exp/remote-config-exp/CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# @firebase/remote-config
+
+## 0.1.28
```

Comment:
I see this file matches master: https://github.com/firebase/firebase-js-sdk/blob/master/packages/remote-config/CHANGELOG.md

When/where will we add a changelog entry for modularization?

Reply:
We will add it before the official release. Actually I shouldn't have included the CHANGELOG.md now. We will create it before the official release.

Deleted.